src/services/ApiService.js

```diff
@@ -2,7 +2,8 @@ import axios from "axios";
 import { handleError } from "../utilities/GlobalFunctions";
 export class Apiservice {
   constructor() {
-    this.baseUrl = process.env.VUE_APP_API_URL;
+    this.baseUrl = "https://bnpl-play-api.herokuapp.com/api/";
```

Comment:
revert this, create .env and paste `VUE_APP_API_URL=https://bnpl-play-api.herokuapp.com/api/`. It should work